src/plugins/telemetry/server/telemetry_collection/get_data_telemetry/constants.ts

```diff
@@ -39,19 +39,24 @@ export const DATA_DATASETS_INDEX_PATTERNS = [
 
   // Observability - Elastic
   { pattern: 'filebeat-*', patternName: 'filebeat', shipper: 'filebeat' },
+  { pattern: '*filebeat*', patternName: 'generic-filebeat' },
   { pattern: 'metricbeat-*', patternName: 'metricbeat', shipper: 'metricbeat' },
+  { pattern: '*metricbeat*', patternName: 'generic-metricbeat' },
   { pattern: 'apm-*', patternName: 'apm', shipper: 'apm' },
   { pattern: 'functionbeat-*', patternName: 'functionbeat', shipper: 'functionbeat' },
+  { pattern: '*functionbeat*', patternName: 'generic-functionbeat' },
   { pattern: 'heartbeat-*', patternName: 'heartbeat', shipper: 'heartbeat' },
+  { pattern: '*heartbeat*', patternName: 'generic-heartbeat' },
   { pattern: 'logstash-*', patternName: 'logstash', shipper: 'logstash' },
+  { pattern: '*logstash*', patternName: 'generic-logstash' },
   // Observability - 3rd party
   { pattern: 'fluentd*', patternName: 'fluentd' },
   { pattern: 'telegraf*', patternName: 'telegraf' },
   { pattern: 'prometheusbeat*', patternName: 'prometheusbeat' },
   { pattern: 'fluentbit*', patternName: 'fluentbit' },
   { pattern: '*nginx*', patternName: 'nginx' },
   { pattern: '*apache*', patternName: 'apache' }, // Already in Security (keeping it in here for documentation)
-  // { pattern: '*logs*', patternName: 'third-party-logs' }, Disabled for now
+  { pattern: '*logs*', patternName: 'third-party-logs' },
```

Comment:
Q: Given that we also catch others like `logstash`, should we call it `generic-logs` instead of 3rd party?

Reply:
I like it!